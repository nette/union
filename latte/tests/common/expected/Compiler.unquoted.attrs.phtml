<?php
%A%
final class Template%a% extends Latte\Runtime\Template
{

	public function main(): array
	{
%A%
		echo '<span title=';
		echo LR\Filters::escapeHtmlAttrUnquoted($x) /* line %d% */;
		echo ' class=';
		echo LR\Filters::escapeHtmlAttrUnquoted($x) /* line %d% */;
		echo '></span>

<span title=';
		echo LR\Filters::escapeHtmlAttrUnquoted($x) /* line %d% */;
		echo ' ';
		echo LR\Filters::escapeHtmlAttrUnquoted($x) /* line %d% */;
		echo '></span>

<span title=';
		if (true) /* line %d% */ {
			echo LR\Filters::escapeHtmlAttrUnquoted($x) /* line %d% */;
		} else /* line %d% */ {
			echo '"item"';
		}
		echo '></span>

<span title=';
		if (true) /* line %d% */ {
			echo LR\Filters::escapeHtmlAttrUnquoted($x) /* line %d% */;
			echo ' ';
			echo LR\Filters::escapeHtmlAttrUnquoted($x) /* line %d% */;
		} else /* line %d% */ {
			echo '"item"';
		}
		echo '></span>

<span ';
		echo LR\Filters::escapeHtmlAttrUnquoted('title') /* line %d% */;
		echo '=';
		echo LR\Filters::escapeHtmlAttrUnquoted($x) /* line %d% */;
		echo '></span>

<span attr';
		echo LR\Filters::escapeHtmlAttrUnquoted($x) /* line %d% */;
		echo 'b=c';
		echo LR\Filters::escapeHtmlAttrUnquoted($x) /* line %d% */;
		echo 'd></span>
';
		return get_defined_vars();
	}

}
