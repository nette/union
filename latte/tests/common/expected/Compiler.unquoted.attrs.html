<span title="&apos; &amp; &quot;" class="&apos; &amp; &quot;"></span>

<span title="&apos; &amp; &quot;" "&apos; &amp; &quot;"></span>

<span title="&apos; &amp; &quot;"></span>

<span title="&apos; &amp; &quot;"></span>

<span attr"&apos; &amp; &quot;"b=c"&apos; &amp; &quot;"d></span>
